src/test/compilation-failed/contracts/const-eval-rawslice-overflow.tact

```diff
@@ -0,0 +1,5 @@
+contract AsciiOverflow {
+    get fun getRawSlice_fail(): Slice {
+        return rawSlice("abcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcd");
```

Comment:
weirdly enough, FunC does not complain about this particular string:
```func
#pragma version =0.4.4;
#pragma allow-post-modification;
#pragma compute-asm-ltr;

_ %foo() method_id(0xff) {
    return "abcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcd"s;
}
```

the code above compiles and produces

```
❯ ./func test.fc
// automatically generated from `test.fc`
255 DECLMETHOD %foo
%foo PROC:<{
  x{abcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcd} PUSHSLICE
}>
```

but if I add one more hex character to the string, then I get an error:

```
test.fc:6:12: error: Invalid hex bitstring constant `abcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcda`
      return "abcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabc ...
             ^
```

Reply:
seems like a FunC problem. when I run the generated fift output - it crashes.

Reply:
Oh, interesting. Can you please file an issue to the monorepo?

Reply:
https://github.com/ton-blockchain/ton/issues/1153